security/src/main/java/com/twilio/security/storage/AuthenticatedEncryptedPreferences.kt

```diff
@@ -153,4 +165,20 @@ class AuthenticatedEncryptedPreferences(
     data: ByteArray,
     kClass: KClass<T>
   ): T? = serializer.fromByteArray(data, kClass)
+
+  private fun getKey(): BiometricSecretKey {
+    if (biometricSecretKey == null) {
+      biometricSecretKey = BiometricSecretKey(
+        AESGCMNoPaddingCipherTemplate(storageAlias, authenticationRequired = true), keyManager
+      )
+    }
+    create()
+    return biometricSecretKey as BiometricSecretKey
+  }
+
+  private fun create() {
```

Comment:
This could be `createIfNeeded` and we can call `biometricSecretKey?.create()` in recreate to prevent a misunderstood about preferences and key status